src/Pdf/Engine/WkHtmlToPdfEngine.php

```diff
@@ -93,6 +93,15 @@ protected function _getCommand()
             throw new Exception(sprintf('wkhtmltopdf binary is not found or not executable: %s', $this->_binary));
         }
 
+        $binaryWrapper = $this->config('binaryWrapper');
+
+        if ($binaryWrapper) {
+            if (!is_executable($binaryWrapper)) {
```

Comment:
I just took that from [line 93](https://github.com/nrother/CakePdf/blob/binarywrapper/src/Pdf/Engine/WkHtmlToPdfEngine.php#L93).
It might be useful to catch spelling errors in the config (file not found).